pdl/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/pdl/Environment.kt

```diff
@@ -1,28 +1,25 @@
 package no.nav.helsearbeidsgiver.inntektsmelding.pdl
 
+import no.nav.helsearbeidsgiver.felles.getEnvVar
+import no.nav.helsearbeidsgiver.felles.oauth2.AzureOAuth2Environment
+
 fun setUpEnvironment(): Environment {
     return Environment(
         raw = System.getenv(),
         pdlUrl = getEnvVar("PDL_URL"),
-        scope = getEnvVar("PROXY_SCOPE"),
-        azureWellKnownUrl = getEnvVar("AZURE_APP_WELL_KNOWN_URL"),
-        azureTokenEndpointUrl = getEnvVar("AZURE_OPENID_CONFIG_TOKEN_ENDPOINT"),
-        azureAppClientID = getEnvVar("AZURE_APP_CLIENT_ID"),
-        azureAppClientSecret = getEnvVar("AZURE_APP_CLIENT_SECRET"),
-        azureAppJwk = getEnvVar("AZURE_APP_JWK")
+        AzureOAuth2Environment(
+            scope = getEnvVar("PROXY_SCOPE"),
+            azureWellKnownUrl = getEnvVar("AZURE_APP_WELL_KNOWN_URL"),
+            azureTokenEndpointUrl = getEnvVar("AZURE_OPENID_CONFIG_TOKEN_ENDPOINT"),
+            azureAppClientID = getEnvVar("AZURE_APP_CLIENT_ID"),
+            azureAppClientSecret = getEnvVar("AZURE_APP_CLIENT_SECRET"),
+            azureAppJwk = getEnvVar("AZURE_APP_JWK")
+        )
```

Comment:
Kunne disse vært default?

Reply:
Kun `AZURE_APP_WELL_KNOWN_URL` og `AZURE_OPENID_CONFIG_TOKEN_ENDPOINT` er samme for hver app.
Så de kan forsåvidt være default